QuizApp/App/Extensions/UIKit/UICollectionView+Extensions.swift

```diff
@@ -0,0 +1,15 @@
+import UIKit
+
+extension UICollectionView {
+
+    static func makeCollectionView(direction: ScrollDirection, spacing: CGFloat) -> UICollectionView {
+        let layout = UICollectionViewFlowLayout()
+        let view = UICollectionView(frame: CGRect.zero, collectionViewLayout: layout)
+        view.showsHorizontalScrollIndicator = false
+        layout.scrollDirection = direction
+        layout.minimumLineSpacing = spacing
+        layout.estimatedItemSize = UICollectionViewFlowLayout.automaticSize
```

Comment:
Can you shorten to `layout.estimatedItemSize = .automaticSize`

Reply:
I did try, but the compiler shows an error `Type 'CGSize' has no member 'automaticSize'`.